polaris-react/src/components/RadioButton/RadioButton.scss

```
@@ -20,30 +24,82 @@
+      --p-color-border-interactive: #1a1a1a;
+      background-color: var(--p-color-bg);
+    }
     border-color: var(--p-color-border-interactive);
-
     // stylelint-disable-next-line selector-max-specificity -- generated by polaris-migrator DO NOT COPY
     &::before {
+      #{$se23} & {
+        // stylelint-disable-next-line -- PSE23 temporary property overrides
+        --p-color-bg-interactive: #fff;
+      }
       transition: opacity var(--p-motion-duration-150) var(--p-motion-ease),
         transform var(--p-motion-duration-150) var(--p-motion-ease);
       opacity: 1;
       transform: translate(-50%, -50%) scale(1);
     }
   }
 
+  #{$se23} &:checked:not([disabled]) + .Backdrop:hover,
+  #{$se23} &:checked:not([disabled]) + .Backdrop.hover {
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-border-interactive: #1a1a1a;
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    &::before {
+      // stylelint-disable-next-line -- PSE23 temporary property overrides
+      --p-color-bg-interactive: rgba(138, 138, 138, 1);
+    }
+  }
+
+  #{$se23} &:disabled + .Backdrop,
+  #{$se23} &:disabled + .Backdrop:hover,
+  #{$se23} &:disabled + .Backdrop.hover {
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-border-disabled: rgba(204, 204, 204, 1);
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-bg-disabled: rgba(204, 204, 204, 1);
+    border-color: var(--p-color-border-disabled);
+  }
+
   &:disabled + .Backdrop {
     border-color: var(--p-color-border-disabled);
     cursor: default;
+    #{$se23} & {
+      background-color: var(--p-color-bg-disabled);
+    }
 
     // stylelint-disable-next-line selector-max-specificity -- generated by polaris-migrator DO NOT COPY
     &::before {
       background-color: var(--p-color-border-disabled);
     }
   }
+  #{$se23} &:disabled:checked + .Backdrop::before,
+  #{$se23} &:disabled:checked + .Backdrop:hover::before,
+  #{$se23} &:disabled:checked + .Backdrop.hover::before {
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-bg-interactive: #fff;
+    background-color: var(--p-color-bg-interactive);
+  }
 }
 
 .Backdrop {
-  // stylelint-disable-next-line -- Polaris component custom properties
+  #{$se23} & {
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-border-input: rgba(138, 138, 138, 1);
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-bg: #fff;
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-color-bg-interactive: #fff;
+    // stylelint-disable-next-line -- PSE23 temporary property overrides
+    --p-border-width-1: 0.66px;
+    border: var(--p-border-width-1) solid var(--p-color-border-input);
+  }
+  // stylelint-disable-next-line -- PSE23 temporary property overrides
```

Looks like this one might have gotten incorrectly reworded?

```suggestion
  // stylelint-disable-next-line -- Polaris component custom properties
```